typings/index.d.ts

```diff
@@ -37,17 +47,21 @@ declare module 'react-digraph' {
     opacity?: number;
     nodeKey: string;
     nodeSize?: number;
+    nodeHeight?: number;
+    nodeWidth?: number;
     onNodeMouseEnter: (event: any, data: any, hovered: boolean) => void;
     onNodeMouseLeave: (event: any, data: any) => void;
     onNodeMove: (point: IPoint, id: string, shiftKey: boolean) => void;
     onNodeSelected: (data: any, id: string, shiftKey: boolean) => void;
     onNodeUpdate: (point: IPoint, id: string, shiftKey: boolean) => void;
+    onOverrideableClick: (event: any) => boolean;
```

Comment:
Would it make sense to name this prop `onNodeClick`?

The return value could be used to set overrideClick logic like you have above, however someone could also use this function to do some action before default behavior is executed (possibly rename return, ex. `preventDefaultNodeInteraction`). Maybe I'm missing something, but just wanted to throw out the idea.